src/main/scala/io/scalac/tezos/translator/routes/MessageRoutes.scala

```diff
@@ -21,10 +22,15 @@ class MessageRoutes(
   import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport._
   import io.circe.generic.auto._
 
+  implicit val ec = actorSystem.dispatcher
```

Comment:
pass execution context as as a parameter of class:

`class MessageRoutes(implicit ec: ...) {`

Reply:
resolved